classes/game.js

```diff
@@ -5,30 +5,54 @@ class Game{
         this.turn = data.turn;
         this.board = new Board(data.board);
         this.me = new Snake(data.you);
+				this.test = data;
     }
     //todo:
     findFood(){
-			let foodx = request.body.board.food[0].x
-			let foody = request.body.board.food[0].y
-			
-			let locx = request.body.you.body[0].x
-			let locy = request.body.you.body[0].y
-			
-			if(locx < foodx){
-				data.move = 'right';
-			}else if(locx > foodx){
-				data.move = 'left';
-			}else if(locy < foody){
-				data.move = 'down';
-			}else if(locy > foody){
-				data.move = 'up';
+			let foodLength = Object.keys(this.test.board.food).length;
```

Comment:
Probably don't need to save a reference the `data` above in `this.test`

Seems like you are just using it to access the board data, so you should be accessing it through `this.board` which already parses the board for us. Don't need two separate references to the same thing

Reply:
Ah thank you. I should've read the constructor properly. I'll get rid of my test placeholder and make the changes